2025/report.md

```diff
@@ -0,0 +1,275 @@
+  - Core: 1
+- [jaq](https://ngi.nixos.org/project/jaq)
+  - Uncategorized: 2
+- [kbin](https://ngi.nixos.org/project/kbin)
+  - Entrust: 2
+- [nyxt](https://ngi.nixos.org/project/nyxt)
+  - Entrust: 1
+  - Review: 2
+- [oku](https://ngi.nixos.org/project/oku)
+  - Entrust: 1
+- [openXC7](https://ngi.nixos.org/project/openXC7)
+  - Entrust: 1
+- [owasp](https://ngi.nixos.org/project/owasp)
+  - Uncategorized: 2
+- [owi](https://ngi.nixos.org/project/owi)
+  - Uncategorized: 2
+- [proximity-matcher](https://ngi.nixos.org/project/proximity-matcher)
+  - Uncategorized: 0
+- [slipshow](https://ngi.nixos.org/project/slipshow)
+  - Commons: 1
+- [stalwart](https://ngi.nixos.org/project/stalwart)
+  - Core: 1
+  - Entrust: 1
+- [verso](https://ngi.nixos.org/project/verso)
+  - Core: 2
+  - Review: 2
+- [xrsh](https://ngi.nixos.org/project/xrsh)
+  - Entrust: 1
+- [y-crdt](https://ngi.nixos.org/project/y-crdt)
+  - Commons: 1
+  - Entrust: 1
+
+### Additinal metrics
+
+#### NGIpkgs repository
+
+NGI team is maintaining
+
+- 47 services and 55 programs
+- 30 projects with demo available
+- 97 NixOS tests, associated with 97 examples
+- 80 derivations, 59 of which have an explicit update script
+
+Funded by following subgrants
+
+- 10 Commons
+- 25 Core
+- 43 Entrust
+- 50 Review
+- 89 Uncategorized
+
+#### Nixpkgs repository
+
+NGI team is maintaining 131 derivations, 107 of which have an explicit update script.
+
+### Non-packaging work
+
+FIXME: do we really need links to PRs ?
+
+In addition to a packaging work, following extra work was done:
+
+- Designed and implemented a new NGI project design (project deliverables) using
+  the NixOS module system
+
+- Implemented [service](https://github.com/ngi-nix/ngipkgs/issues/614) and
+  [program](https://github.com/ngi-nix/ngipkgs/issues/1366) demos as a practical
+  way to showcase projects
+
+- [Added update script support and bulk updates](https://github.com/ngi-nix/summer-of-nix/issues/121)
+
+- Improved project and deliverable tracking with
+  [issue templates](https://github.com/ngi-nix/ngipkgs/issues/new/choose)
+
+- Implemented multiple improvements to the UI/UX of the project
+  [overview page](https://ngi.nixos.org/) such as:
+  - implementation of website generation using NixOS modules
+  - implementation of metadata, deliverables, demo and demo instructions sections
+  - implementation of copy-paste buttons
+  - implementation of update script status
+  - implementation of highlighting of missing programs, services, links or deliverables
+  - implementation of breadcrumbs
+  - addition of usage instructions for multiple Linux distributions
+  - implementation of dark mode
+
+- Signifficant improvements of tasks and project management
+  [tools](https://github.com/orgs/ngi-nix/projects/8)
+  and processes (planning, backlog grooming and reivew sessions)
+
+- Provided support for integration of Nix based development environments to
+  upstream source code bases of NLnet funded sofware
```

Comment:
We can perhaps give an example for this, the most recent being the [dev env for 0WM](https://github.com/eljamm/0WM-Server/tree/feat/nix-dev).

Reply:
I can't see flake.nix merged to 0WM upstream yet. Is it ?